src/ripple/app/paths/impl/Steps.h

```diff
@@ -245,9 +246,11 @@ toStrand (
     AccountID const& src,
     AccountID const& dst,
     Issue const& deliver,
+    boost::optional<Quality> const& limitQuality,
```

Comment:
This `limitQuality` is different from the payment `limitQuality`, and that's a potential source of confusion.